src/js/tabs/setup.js

```diff
@@ -416,35 +415,58 @@ setup.initialize = function (callback) {
             }
         };
 
+        function showDialogBuildInfo(title, message) {
+            const dialog = $('.dialogBuildInfo')[0];
+
+            $('.dialogBuildInfo-title').html(title);
+            $('.dialogBuildInfo-content').html(message);
+
+            if ( ! dialog.hasAttribute('open')) {
+                dialog.showModal();
+                $('.dialogBuildInfo-closebtn').on('click', function() {
+                    dialog.close();
+                });
+            }
+        }
+
         const showFirmwareInfo = function() {
             // Firmware info
             msp_api_e.text([FC.CONFIG.apiVersion]);
             build_date_e.text([FC.CONFIG.buildInfo]);
 
-            if (FC.CONFIG.buildKey.length === 32) {
-                const buildRoot   = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
-                const buildConfig = `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildInfoConfig')}: ${buildRoot}/json">
-                                     <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildInfoConfig')}</a></strong></span>`;
-                const buildLog =    `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildInfoLog')}: ${buildRoot}/log">
-                                     <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildInfoLog')}</a></strong></span>`;
-                build_info_e.html(`${buildConfig} &nbsp &nbsp ${buildLog}`);
-                $('.build-info a').removeClass('disabled');
-            } else {
-                $('.build-info a').addClass('disabled');
-            }
+            if (navigator.onLine) {
+                let buildOptionList = "";
 
-            if (FC.CONFIG.buildOptions.length) {
-                let buildOptions = "";
-                build_opt_e.text = "";
+                if (FC.CONFIG.buildOptions.length) {
 
-                for (const buildOption of FC.CONFIG.buildOptions) {
-                    buildOptions = `${buildOptions} &nbsp ${buildOption}`;
+                    for (const buildOptionElement of FC.CONFIG.buildOptions) {
+                        buildOptionList = `${buildOptionList} &nbsp ${buildOptionElement}`;
```

Comment:
```suggestion
                        buildOptionList = `${buildOptionList} ${buildOptionElement}`;
```

would present it like
![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/41b7401b-964b-40f2-9b06-419bce880edf)

Doing 3 x 3 would be even better. Could use  `.padEnd()`

Reply:
Hmm no .padEnd() does not seem to work with html as it removes the spaces :(

Reply:
I try some loops

Reply:
Might need css - flex columns for that as fonts are disproportional.